### PR TITLE
Updated the list of required Jekyll theme gems.

### PR DESCRIPTION
**Pull request by kylekirkby**
Updated the list of required Jekyll theme gems.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -124,15 +124,14 @@ ENV RUBY_GEMS \
...
- jumbo-jekyll-theme:4.7.7 \
- # Used by connect, ai
+ # Used by staging.linaro.org
+ jumbo-jekyll-theme:5.5.4 \
+ # Used by ai
  jumbo-jekyll-theme:5.3.4 \
- # Used by OP-TEE, Linaro.cloud
+ # Used by 96boards.org
+ jumbo-jekyll-theme:5.5.3
+ # Used by op-tee.org, linaro.cloud, linaro.org, devicetree.org,
```
Comment (morancj) [inline Dockerfile]: Alpha-sort?

Comment (morancj) [inline Dockerfile]:
```suggestion
 # Used by connect.linaro.org, devicetree.org, linaro.cloud, linaro.org,
```

File: Dockerfile [hunk 2]
```diff
@@ -124,15 +124,14 @@ ENV RUBY_GEMS \
...
+ # connect.linaro.org, trustedfirmware.org, mlplatform.org
```
Comment (morancj) [inline Dockerfile]:
```suggestion
 # mlplatform.org, op-tee.org, trustedfirmware.org
```

File: Dockerfile [hunk 3]
```diff
@@ -124,15 +124,14 @@ ENV RUBY_GEMS \
  jumbo-jekyll-theme:1.4.4 \
  # Used by ?
  jumbo-jekyll-theme:3.9.4 \
- # Used by ArmNN
- jumbo-jekyll-theme:4.4.4 \
- # Used by devicetree, TrustedFirmware
- jumbo-jekyll-theme:4.5.0 \
- # Used by linaro.org
- jumbo-jekyll-theme:4.7.7 \
- # Used by connect, ai
```
Comment (morancj) [inline Dockerfile:133]: To which sites did this refer? [connect.linaro.org](https://connect.linaro.org) and [www.96boards.ai](https://www.96boards.ai)?

File: Dockerfile (hunk 1 above)
Comment (morancj) [reply Dockerfile]: Missing `, www.96boards.ai`?

**Review by morancj — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 3 above)
Comment (kylekirkby) [reply Dockerfile:133]: connect = connect.linaro.org 
ai = 96boards.ai

Comment (morancj) [reply Dockerfile:133]: Let's use the full names to ease discovery :smile:

**Review by morancj — APPROVED** [review]

File: Dockerfile [hunk 4]
```diff
@@ -107,38 +107,37 @@ ENV RUBY_GEMS \
...
+ # Used by staging.linaro.org
+ jumbo-jekyll-theme:5.5.4 \
+ # Used by staging.96boards.ai
  jumbo-jekyll-theme:5.3.4 \
- # Used by OP-TEE, Linaro.cloud
+ # Used by 96boards.org
+ jumbo-jekyll-theme:5.5.3
```
Comment (pcolmer) [inline Dockerfile]: This needs a trailing \

**Review by pcolmer — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 4 above)
Comment (kylekirkby) [reply Dockerfile]: Done :)

**Review by pcolmer — APPROVED** [review]

**Pull request merged**